src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -401,15 +402,20 @@ private function modifyAllLayers(ImageInterface $image, callable $modifier)
 
         if (count($layers)) {
             $countLayer = 0;
-            $image->layers()->coalesce();
+
+            // coalesce currently not supported by the VipsAdapter:
+            // see: https://github.com/rokka-io/imagine-vips/issues/
+            if (!$image instanceof VipsImage) {
```

Comment:
@alexander-schranz Didn't we say that the `coalesce` function should be a noop in the `Vips` adapter, because it seems it is not necessary? If the maintainers agree and would make a release soonish, I would be more comfortable with waiting until this is fixed before merging this, instead of having this `if` with a reference to a concrete adapter...